docs/installation.md

```diff
@@ -0,0 +1,126 @@
+# Installation
+This is a guide to building and deploying Shesmu.
+
+## Dependencies
+What Shesmu requires will depend on which plugins you enable. Plugins can be
+disabled by passing `-pl "!plugin-foo"` to disable a plugin when calling `mvn`
+commands.
+
+Build dependencies:
+
+- Java 17 or later
+- Maven 3.8 or later
+- NPM
+- Docker (optional) for container builds
+
+Optional runtime dependencies:
+
+- Prometheus (strongly recommended)
+- Prometheus Alert Manager (required for `Olive Alert` to work)
+- JIRA
+- GitHub
+- GitHub, GitLabs, or BitBucket for storing configuration files (recommended)
```

Comment:
GitLabs, plural?

Reply:
Also Github is here twice

Reply:
They're doing different things, so updated.